hivemq-edge/src/main/java/com/hivemq/api/model/TagResourceExamples.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2019-present HiveMQ GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hivemq.api.model;
+
+public class TagResourceExamples {
+
+    public static final String EXAMPLE_OPC_UA = "{\n" +
```

Comment:
Either use a statix ObjectMapper or have a test for these.
Reason is that we might change the API and people won't catch the strings in here, breaking our docs.